converter/src/main/java/gov/cms/qpp/conversion/PathSource.java

```diff
@@ -11,10 +11,20 @@
  */
 public class PathSource extends SkeletalSource {
 
+	private static final String fileName(Path path) {
+		if (path != null) {
+			Path fileName = path.getFileName();
+			if (fileName != null) {
+				return fileName.toString();
+			}
+		}
+		return "";
+	}
+
 	private final Path path;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1213)